ivy/container/general.py

```diff
@@ -50,3 +51,106 @@ def clip_vector_norm(
             ),
             out,
         )
+
+    def all_equal(
+        self,
+        x,
+        equality_matrix: bool = False
+    ):
+        """Determines whether the inputs are all equal.
```

Comment:
The docstring before the examples need not have the description of the parameters and what's returned, you can refer [this](https://github.com/unifyai/ivy/blob/master/ivy/container/elementwise.py#L139) docstring for the same. Similarly, you can refer [this](https://github.com/unifyai/ivy/blob/master/ivy/container/elementwise.py#L82) docstring to see how to add the docstring for the static function you'll be adding.